src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -294,6 +294,12 @@ function generateAdditionalConditions(
           ? String(value)
           : String(value).trim();
 
+      if (column.field.startsWith('measurements.') && !nextValue) {
```

Comment:
I'd recomment using `isMeasurement` from `src/sentry/static/sentry/app/utils/discover/fields.tsx` to do this check.

Reply:
Should be addressed in https://github.com/getsentry/sentry/pull/21905/commits/0d76582eaaf379f279a795b5004591e90096da7c